ivy.xml

```diff
@@ -0,0 +1,34 @@
+<ivy-module version="2.0">
+    <info organisation="com.google.enterprise" module="adaptors-library"/>
+
+    <configurations>
+      <conf name="compile" description="Required to compile application"/>
+      <conf name="runtime" description="Additional run-time dependencies"/>
+      <conf name="test" description="Test-only dependencies"/>
+      <conf name="misc" description="Misc components' dependencies (tools, examples)"/>
+      <conf name="buildtools" description="Ant build dependencies"/>
+    </configurations>
+
+    <dependencies>
+        <dependency org="com.googlecode.json-simple" name="json-simple" rev="1.1" conf="compile->default"/>
+        <dependency org="com.google.code.findbugs" name="jsr305" rev="3.0.2" conf="compile->default"/>
+        <dependency org="com.google.guava" name="guava" rev="17.0" conf="compile->default"/> <!-- 17.0 (newest: 25.0-jre) -->
```

Comment:
I updated some libraries to newer versions, where it didn't break anything. the comment contains the original versions from lib/ folder. The newest 25.0-jre guava broke the build (easily fixable).